scipy/stats/stats.py

```diff
@@ -2882,6 +2901,13 @@ def median_abs_deviation(x, axis=0, center=np.median, scale=1.0,
     will calculate the MAD around the mean - it will not calculate the *mean*
     absolute deviation.
 
+    The default `center` function is `numpy.median`, but it is wrapped in
+    code that will prevent it from generating a warning if it is given an
```

Comment:
hmm, maybe I'm just not seeing it, but where is the wrapped in code that prevents generating a warning, is it already in `median_abs_deviation` and unchanged here so not being displayed?

Reply:
Thanks for catching that.  That note is out of date.  An earlier version of my changes had an explicit check for `np.median`, but now that is gone.  I'll remove the note.

Reply:
looks good in latest